### ISSUE TITLE
Disable Gamepad detection in options

### ISSUE THREAD

**Issue by nirox** [JosefNemec/Playnite#2224]
**Check for existing issue**
When you try to use the app in VR, the knuckles are detected wrong in some devices.

**Describe the solution you'd like**
If you develop a option to disable gamepad detection, I think that this resolve the problem, because you can use the knuckles like a mouse and keyboard

**Screenshots**
--

**Comment by Jlll1** [comment]
Is there anyone working on it? I'd like to pick that up.

**Comment by JosefNemec** [comment]
I don't think so, but I would suggest waiting for Playnite 9 to be finished, because the whole Fullscreen settings section will be completely reworked.

**Comment by Jlll1** [comment]
Sure. Thanks for answering.
